### PR TITLE
DO-295: New theme added.

### PR DESCRIPTION
**Pull request by amitjain24524**

### CODE REVIEW

File: src/components/common/AppFooter/AppFooterDesktop/_AppFooterDesktop.scss [hunk 1]
```diff
@@ -10,15 +10,18 @@
...
   @media (max-width: 767px) {
-    height: 26px;
+    height: 32px;
   }
   .app-footer__empty-bg {
     height: 4px;
     width: 100%;
     display: block;
-    background: linear-gradient(45deg, var(--secondary-color) , var(--primary-color));
+    background: linear-gradient(45deg, var(--primary-color) , var(--primary-color));
```
Comment (pepcus-ankit) [inline src/components/common/AppFooter/AppFooterDesktop/_AppFooterDesktop.scss:19]: Is it correct to have gradient of same color?

File: src/components/common/AppHeader/AppHeaderDesktop/_AppHeaderDesktop.scss [hunk 2]
```diff
@@ -28,11 +28,12 @@
   display: flex;
   align-items: center;
   flex-wrap: wrap;
-  background-image: linear-gradient(to right, var(--secondary-color),  var(--primary-color));
+  background-image: linear-gradient(to right, var(--primary-color),  var(--primary-color));
```
Comment (pepcus-ankit) [inline src/components/common/AppHeader/AppHeaderDesktop/_AppHeaderDesktop.scss:31]: Again gradient of same color

File: src/components/routes/Member/AuthenticatedMember/UpdateCredentialsRoute/UpdateCredentialsContent/UpdateUsernameForm/UpdateUsernameForm.jsx [hunk 3]
```diff
@@ -55,7 +55,7 @@ const UpdateUsernameForm = (props) => {
 
   const renderSubmitButton = () => (
     <div className="update-password__button-container">
-      <Button className="button__secondary-color update-password_button" type="submit" value="Update" />
+      <Button className="button button button__small button__blue" type="submit" value="Update" />
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/UpdateCredentialsRoute/UpdateCredentialsContent/UpdateUsernameForm/UpdateUsernameForm.jsx:58]: why the same button class used twice?
```suggestion
      <Button className="button button__small button__blue" type="submit" value="Update" />
```

Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/UpdateCredentialsRoute/UpdateCredentialsContent/UpdateUsernameForm/UpdateUsernameForm.jsx]: done

File: src/components/common/AppHeader/AppHeaderDesktop/_AppHeaderDesktop.scss (hunk 2 above)
Comment (amitjain24524) [reply src/components/common/AppHeader/AppHeaderDesktop/_AppHeaderDesktop.scss]: Done

File: src/components/common/AppFooter/AppFooterDesktop/_AppFooterDesktop.scss (hunk 1 above)
Comment (amitjain24524) [reply src/components/common/AppFooter/AppFooterDesktop/_AppFooterDesktop.scss]: Done

**Pull request merged**